### PR TITLE
fix crypto share

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
This PR fixes the go-client method to ListKeys (only one email was set)
Fixes edge case when there are no keys
 

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [X] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: pkg/api/users.go [hunk 1]
```diff
@@ -135,9 +135,9 @@ func (client *client) GrabAccessToken() (string, error) {
 }
 
 func (client *client) ListKeys(emails []string) ([]*PublicKey, error) {
-	emailsInput := make([]*string, 0)
-	for _, email := range emails {
-		emailsInput = append(emailsInput, &email)
+	emailsInput := make([]*string, len(emails))
```
Comment (michaeljguarino) [inline pkg/api/users.go:138]: use `lo.ToSlicePtr` here

Comment (zreigz) [reply pkg/api/users.go]: done

**Review by michaeljguarino — APPROVED** [review]
